### ISSUE TITLE
bug: change date of the feast of Our Lord Jesus Christ, The Eternal High Priest

### ISSUE THREAD

**Issue by tukusejssirs** [romcal/romcal#25]
As it is noted [here](https://en.wikipedia.org/wiki/Feast_of_Christ_the_Priest), this feast is to be celebrated **on the first Thursday after Pentecost** in countries, where it was requested by local episcopal conference and approved/granted by the Congregation for Divine Worship and the Discipline of the Sacraments.

Another thing is the feast’s name. In English, I’ve found two different name for (in Latin) _festum Domini nostri Iesu Christi Summi et Æterni Sacerdotis_:

(1) Our Lord Jesus Christ, Supreme and Eternal Priest;
(2) Our Lord Jesus Christ, the Eternal High Priest,

while the latter I’ve found more times and even you used this one. But the former better translated its Latin name. 

**Edit:**
Here’s a list of countries and the dates you coded in (and request an addition of it into `czechRepublic.mjs`:
```bash
$ grep -inA2 ourLordJesusChristTheEternalHighPriest ../calendars/*
../calendars/bolivia.mjs:164:      "key": "ourLordJesusChristTheEternalHighPriest",
../calendars/bolivia.mjs-165-      "type": Types[4],
../calendars/bolivia.mjs-166-      "moment": ( y => Dates.pentecostSunday( y ).add( 4, 'days' ))(year),
--
../calendars/mexico.mjs:115:      "key": "ourLordJesusChristTheEternalHighPriest",
../calendars/mexico.mjs-116-      "type": Types[4],
../calendars/mexico.mjs-117-      "moment": moment.utc({ year: year, month: 5, day: 16 }),
--
../calendars/peru.mjs:77:      "key": "ourLordJesusChristTheEternalHighPriest",
../calendars/peru.mjs-78-      "type": Types[4],
../calendars/peru.mjs-79-      "moment": moment.utc({ year: year, month: 5, day: 16 }),
--
../calendars/poland.mjs:715:      "key": "ourLordJesusChristTheEternalHighPriest",
../calendars/poland.mjs-716-      "type": Types[4],
../calendars/poland.mjs-717-      "moment": moment.utc({ year: year, month: 5, day: 16 }),
--
../calendars/puertoRico.mjs:99:      "key": "ourLordJesusChristTheEternalHighPriest",
../calendars/puertoRico.mjs-100-      "type": Types[4],
../calendars/puertoRico.mjs-101-      "moment": moment.utc({ year: year, month: 5, day: 16 }),
--
../calendars/slovakia.mjs:210:      "key": "ourLordJesusChristTheEternalHighPriest",
../calendars/slovakia.mjs-211-      "type": Types[4],
../calendars/slovakia.mjs-212-      "moment": moment.utc({ year: year, month: 5, day: 16 }),
```

**Comment by tukusejssirs** [comment]
Besides Czech Republic, this Feast is also celebrated is Netherlands and Australia.

**Comment by tukusejssirs** [comment]
This is still a bug.

We need to remove `jesusChristTheEternalHighPriest` and `ourLordJesusChristTheEternalHighPriest` from all calendars except for `general.js`, where `ourLordJesusChristTheEternalHighPriest` should be added.

**Comment by emagnier** [comment]
This is a celebration (and calendar) issue, not a bug. I've added this new label to make a better difference in the issue tracker between code / romcal bugs and issues directly related to a celebration (I made the same for everything related to localizations).

**Comment by tukusejssirs** [comment]
Okay. I don’t really if we label this as bug or whatever else, but should create a list of labels with their description that would include the use cases for that particular label.

**Comment by emagnier** [comment]
Labels have already their description (tooltip or in the label picker) when the name might be not enough to understand the goal of a label.

I added theses 2 new labels to visualize better the issue tracker, sort easily the type of work, and improve the project management in general.

**Comment by jarosz** [comment]
> We need to remove `jesusChristTheEternalHighPriest` and `ourLordJesusChristTheEternalHighPriest` from all calendars except for `general.js`, where `ourLordJesusChristTheEternalHighPriest` should be added.

I don't agree. According to the rubrics it's not in the general calendar and therefore may have different dates and names in particular calendars.

**Comment by tukusejssirs** [comment]
@jarosz, you’re right. :smiley:

I have just found out that it is wrongly defined in `slovakia.js` only, so I’ll fix it. Also, I’ll add the missing celebration in some of calendars (listed in one of my previous comments).

**Comment by tukusejssirs** [comment]
@emagnier, have you fixed this issue already? :smiley:

**Comment by emagnier** [comment]
I guess you fixed it already: #120.
